sourced/ml/transformers/basic.py

```diff
@@ -225,13 +237,18 @@ def create_parquet_loader(session_name, repositories,
     return parquet
 
 
-def create_uast_source(args, session_name, select=HeadFiles):
+def create_uast_source(args, session_name, select=HeadFiles, language_selector=None,
+                       extract_uast=True):
     if args.parquet:
         start_point = create_parquet_loader(session_name, **args.__dict__)
         root = start_point
```

Comment:
I think it should check the availability of `uast`  field according to argument `extract_uast`.
You can receive a list of column names like this `root.schema.names` - it will allow debugging errors faster and easier.